### DISCUSSION TITLE
Variable font size for the suggestions and chat panels

### DISCUSSION THREAD

**Discussion by KieranConlon** [intitni/CopilotForXcode#176] [Ideas]
The font size used for the suggestions and chat panels appears to be fixed size.
Could having a variable font size be a nice improvement?

Maybe the font size could mirror font settings in Xcode - although I'm not sure if this information is available programmatically.

Alternatively, could we create custom commands to increase/decrease the font size?  These custom commands could then have key bindings assigned.

**Comment by intitni** [comment]
The change font size feature is now included in 0.14.0 beta 4.

**Reply by j-peace** [reply]
Hi, there are more relevant features available just after 0.14.0 ?
